src/bin/stratis_uuids_to_names.rs

```diff
@@ -42,8 +43,14 @@ const STRATIS_MANAGER_OBJECT: &str = "/org/storage/stratis2";
 const STRATIS_POOL_IFACE: &str = "org.storage.stratis2.pool.r1";
 const STRATIS_FS_IFACE: &str = "org.storage.stratis2.filesystem";
 const DBUS_OM_IFACE: &str = "org.freedesktop.DBus.ObjectManager";
+
 lazy_static! {
-    static ref TIMEOUT: Duration = Duration::new(5, 0);
+    // The value is 2**31 - 1 millisecond, the largest representable value
+    // in a 32-bit signed int. -1, which is often used to indicate something
+    // unbounded is interpreted by libdbus as a request for its default timeout,
+    // which we consider unacceptably small.
+    static ref TIMEOUT: Duration =
+        Duration::from_millis(TryFrom::try_from(std::i32::MAX).expect("statically verified"));
```

Comment:
`i32::MAX` does indeed mean infinite to libdbus, so I vote for naming this constant `DBUS_TIMEOUT_INFINITE`, to match libdbus. See https://dbus.freedesktop.org/doc/api/html/group__DBusPendingCall.html#ga21384c9c5b0da54f7d0a92012522f213.

(For my own education: I am surprised at all the ceremony; why not just `Duration::from_millis(std::i32::MAX)`?)

Reply:
```Duration::from_millis()```'s parameter type is u64, Rust's type system requires a conversion.